focus-blocked/js/locale.js

```diff
@@ -70,6 +70,8 @@ const locale = {
       `Let's keep the focus on ${focus_mode}`,
     get_a_tip_for_staying_focused: 'Get a tip for staying focused',
     original_url: (url) => `Original URL ${url}`,
+    general_error:
+      'Something went wrong while creating the course. Very sorry. Please email support@focusbear.io and tell us what led to this error',
```

Comment:
Is this the correct error message for the blocked page?

Reply:
sorry I will fix it asap

Reply:
![image](https://github.com/Focus-Bear/assets/assets/11420548/57cf2e86-16b2-4bc0-b012-d31b4d89c236)